sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/TelemetryExceptionData.cs

```diff
@@ -15,7 +15,11 @@ internal partial class TelemetryExceptionData
 
         public TelemetryExceptionData(int version, LogRecord logRecord) : base(version)
         {
-            var message = LogsHelper.GetMessage(logRecord);
+            Properties = new ChangeTrackingDictionary<string, string>();
+            Measurements = new ChangeTrackingDictionary<string, double>();
```

Comment:
used anywhere?

Reply:
Swagger schema mandates us to use this property. This is not used anywhere, but needs to initialized for serialization to succeed. Swagger update is needed to remove this from all the languages.